src/Transport/Topology/Topologies/EndpointOrientedTopologySectionManager.cs

```diff
@@ -193,39 +193,81 @@ TopologySectionInternal BuildSubscriptionHierarchy(Type eventType)
 
             var topics = new List<EntityInfoInternal>();
             var subs = new List<SubscriptionInfoInternal>();
-            foreach (var topicPath in topicPaths)
+            foreach (var publisher in publishers)
             {
+                var topicPath = publisher + ".events";
                 var path = addressingLogic.Apply(topicPath, EntityType.Topic).Name;
-                topics.AddRange(namespaces.Select(ns => new EntityInfoInternal
+
+                var destinationsOutsideTopology = namespaceConfigurations.Where(c => c.RegisteredEndpoints.Contains(publisher, StringComparer.OrdinalIgnoreCase)).ToList();
```

Comment:
`destinationsOutsideTopology` - "destinations" or "subscribers" outside of topology?